README.md

```diff
@@ -96,6 +98,17 @@ To authenticate follow the steps:
 5. Close the browser window and return to VS Code.
    VS Code is now reading and saving the authentication on your local machine.
 
+### Switching accounts
+
+To re-authenticate with a different account, follow the steps below:
+
+1. Run the provided `Snyk: Logout` command:
```

Comment:
Not a grammar nazi here, but log out is an action verb, whereas logout is a noun. I believe we should use the former :)

Also is logout the right word? I know sign out is used by VS Code internally. We can have it consistent. 
<img width="448" alt="Screenshot 2022-03-21 at 14 28 17" src="https://user-images.githubusercontent.com/2239563/159270733-896373bf-a948-4194-888c-5be131e9c4d6.png">

Reply:
😄 
I don't have a preference tbf. I saw that we have a `Login` method maybe to tie that down with a `Logout`. Happy to change it to `Sign Out`. :)